vue/components/organisms/pickers/pickers.vue

```diff
@@ -396,6 +400,7 @@ body.mobile .button-scroll-materials {
     left: 0px;
 }
 
+body.tablet .button-scroll-colors,

```

Comment:
are we sure about this? I believe the pickers in tablet are pretty much the same as in desktop? @gcandal can you confirm?

Reply:
It resulted from this difference when the `white-pickers` style is removed:<br>
Without change:
![Screenshot_2020-12-07 RIPE White(1)](https://user-images.githubusercontent.com/25725586/101337098-2fd7d300-3873-11eb-838d-940ce4cbcb69.png)
With change:
![Screenshot_2020-12-07 RIPE White](https://user-images.githubusercontent.com/25725586/101337105-31090000-3873-11eb-8004-645d7b611991.png)

Reply:
@joamag there's a variation in the pickers: in tablet there is no close button in the pickers (`X`)